provisioning/miniprov/hare_mp/main.py

```diff
@@ -389,10 +401,27 @@ def start_mkfs(hostname: str, hare_config_dir: str):
                           kv_file=f'{hare_config_dir}/consul-kv.json')
     cmd = '/usr/libexec/cortx-motr/motr-mkfs'
     # start mkfs for confd, ios services
+    start = perf_counter()
+    cmd_list = []
     for svc in ('confd', 'ios'):
         svc_fids = generator.get_svc_fids(svc)
         for fid in svc_fids:
-            execute([cmd, fid, '--conf'])
+            # real command line
+            logging.info(f'fid = {str(fid)}\n')
+            cmd_line = [cmd, fid, '--conf']
+            cmd_list += [cmd_line]
+
+    ret_values = []
+    with concurrent.futures.ProcessPoolExecutor() as executor:
+        ret_values = list(executor.map(start_mkfs, cmd_list))
+    finish = perf_counter()
+    perf_result = str(finish - start)
+
+    if not all(rc == 0 for rc in ret_values):
+        logging.error('Error during mkfs start')
```

Comment:
Don't we need to raise an exception here?

The aim is to make sure that the exit code is not 0 if there was a problem during the execution.

Reply:
Done